packages/gatsby/index.d.ts

```diff
@@ -1174,20 +1313,24 @@ export interface Reporter {
 }
 
 export interface Cache {
```

Comment:
This interface wasn't being used at all, so I don't think it's likely that this will cause a breaking change.

Reply:
hmm. technically it is a breaking change as consumers of gatsby that depend on this it's changing.

But I'm curious why this is changing so much?

Reply:
I believe the typing is incorrect and exposes too many functions that are not part of the public Cache API: https://www.gatsbyjs.org/docs/node-api-helpers/#GatsbyCache

If you're concerned about backwards compatibility, I can leave this here and make a new `GatsbyCache` interface instead with the public API.